vignettes/diving_deeper.Rmd

```diff
@@ -106,9 +108,9 @@ The **granularity** of each table (i.e. the observational unit that a single row
 |                  |                          |                             | `redcap_arm`                |
 +------------------+--------------------------+-----------------------------+-----------------------------+
 
-> Note: Taken in combination, the identifying columns of any REDCapTidieR tibble are guaranteed to be unique and not `NA` and can therefore be used as **composite primary key**. This makes it easy to join REDCapTidieR tibbles to each other!
+> Note: Taken in combination, the identifier columns of any REDCapTidieR tibble are guaranteed to be unique and not `NA` and can therefore be used as **composite primary key**. This makes it easy to join REDCapTidieR tibbles to each other!
 
-Let's look at a REDCap database you might use to capture data for a clinical trial. This is a longitudinal database that assesses some data once, such as demographics, and other data, such as a physical exam or chemistry laboratory values, at multiple pre-defined study visits. We are using `dplyr::select` here to highlight instrument names and whether or not each instrument is repeating. This database has six nonrepeating and three repeating instruments.
+Let's look at a REDCap database you might use to capture data for a clinical trial. This is a longitudinal database that assesses some data, such as demographics, just once at enrollment, and other data, such as a physical exam or chemistry laboratory values, multiple times at pre-defined study visits. We use `dplyr::select()` here to highlight instrument names and whether or not each instrument is repeating. This database has six nonrepeating and three repeating instruments.
```

Comment:
> ... "some data, such as demographics, just once at enrollment, and other data, such as physical exam or chemistry laboratory values," ...

Because everything's separated by commas this takes a few reads to understand what's connected where. If you want to keep all of these I would put them in parentheses to better identify:

> ... "some data, such as demographics (taken just once at enrollment), and other data (such as physical exam or chemistry laboratory values)," ...

Reply:
Done!